amundsen_gremlin/gremlin_model.py

```diff
@@ -0,0 +1,492 @@
+        label='Source',
+        properties=[])
+    Stat = VertexType.construct_type(
+        label='Stat',
+        properties=[
+            Property(name='stat_val', type=GremlinType.String),
+            Property(name='stat_type', type=GremlinType.String, comment='effectively an enum'),
+            Property(name='start_epoch', type=GremlinType.Date),
+            Property(name='end_epoch', type=GremlinType.Date)])
+    Table = VertexType.construct_type(
+        label='Table',
+        properties=[
+            Property(name='name', type=GremlinType.String, required=True),
+            Property(name='is_view', type=GremlinType.Boolean),
+            Property(name='display_name', type=GremlinType.String)])
+    Tag = VertexType.construct_type(
+        label='Tag',
+        properties=[
+            Property(name='tag_name', type=GremlinType.String, required=True),
+            Property(name='tag_type', type=GremlinType.String, required=True, default='default',
+                     comment='effectively an enum, usually default')])
+    Timestamp = VertexType.construct_type(
+        label='Timestamp',
+        properties=[])
+    Updatedtimestamp = VertexType.construct_type(
+        label='Updatedtimestamp',
+        properties=[
+            Property(name='latest_timestamp', type=GremlinType.Date, required=True)])
+    User = VertexType.construct_type(
+        label='User',
+        properties=[
+            Property(name='user_id', type=GremlinType.String, required=True),
+            Property(name='email', type=GremlinType.String),
+            Property(name='full_name', type=GremlinType.String),
+            Property(name='first_name', type=GremlinType.String),
+            Property(name='last_name', type=GremlinType.String),
+            Property(name='display_name', type=GremlinType.String),
+            Property(name='team_name', type=GremlinType.String),
+            Property(name='employee_type', type=GremlinType.String, comment='this is effectively an enum'),
+            Property(name='is_active', type=GremlinType.Boolean),
+            Property(name='profile_url', type=GremlinType.String),
+            Property(name='role_name', type=GremlinType.String),
+            Property(name='slack_id', type=GremlinType.String),
+            Property(name='github_username', type=GremlinType.String),
+            Property(name='manager_fullname', type=GremlinType.String),
+            Property(name='manager_email', type=GremlinType.String),
+            Property(name='manager_id', type=GremlinType.String,
+                     comment='the key/user_id of another User who is the manager for this User'),
+            Property(name='is_robot', type=GremlinType.Boolean)
+        ])
+    Watermark = VertexType.construct_type(
+        label='Watermark',
+        properties=[])
+
+
+class EdgeTypes(Enum):
+    """
+    In general, you will need to reload all your data: 1. change label, 2. if you change the type of a property,
```

Comment:
same comment RE: this comment missing a couple of words.
```suggestion
    In general, you will need to reload all your data if you: 1. change label, 2. if you change the type of a property,
```